ui/src/config.js

```diff
@@ -49,3 +49,8 @@ export const sentryConfig = {
 export const monitoringConfig = {
   dashboardUrl: process.env.REACT_APP_ROUTER_MONITORING_URL || ""
 };
+
+export const alertConfig = {
+  environment:
+    appConfig.environment === "dev" ? "development" : appConfig.environment
```

Comment:
I think we should not have this hardcoded environment name conversion logic in the codebase, should just use `appconfig.environment`. Any name conversion from dev to development (or prod to production, p to production etc) should probably be done separately, as these naming conventions usually are specific to diferent organisation/users. We can add a TODO comment or issue for this.